lib/worker.js

```diff
@@ -13,6 +13,12 @@ module.exports = async function ({ targets }) {
   targets = await Promise.all(targets.map(async (t) => {
     let fn
     try {
+      if(typeof process !== "undefined"){
+        //reuse the error code.
```

Comment:
What is this comment referencing?

Reply:
I am sorry. I meant that I reused the error code for the condition to match in the if statement catch clause.